lib/rules/no-dynamic-import.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+'use strict';
+
+const { docUrl } = require('../util/doc-url');
+
+module.exports = {
+    meta: {
+        docs: {
+            description: 'disallow dynamic import use.',
+            category: 'Locker',
+            url: docUrl('no-dynamic-import'),
+        },
+        schema: [],
+    },
+
+    create(context) {
+        return {
+            CallExpression(node) {
```

Comment:
`import()` is a reserved keyword and is not treated like any other call expression from a parsing perspective. It's an `ImportExpression`.

Reply:
Ah nice yep! I'm stuck in pre-implemented brain (used to implementing it myself). Fixing :)